doc/operations/ledger_snapshot.rst

```diff
@@ -86,3 +86,18 @@ A low value for ``ledger.chunk_size`` means that smaller ledger files are genera
 Similarly, a low value for ``snapshots.tx_count`` means that snapshots are generated often and that join/recovery time will be short, at the cost of additional workload on the primary node for snapshot generation.
 
 .. tip:: Uncommitted ledger files (which are likely to contain committed transactions) can also be used on join/recovery, as long as they are copied to the node's ``ledger.directory`` directory.
```

Comment:
Should we start adjusting that to exclude temporary recovery files?

Reply:
Yes, as soon as the feature is on `main`, we should reflect this here.